internal/storage/v2/grpc/proto/traces_storage.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
```

Comment:
I don't think we need to have a folder for this file

```diff
- internal/storage/v2/grpc/proto/traces_storage.proto
+ internal/storage/v2/grpc/trace_storage.proto
```

also, we use singular `tracestorage` in the package name

Reply:
@yurishkuro is it fine for these files to sit with the storage implementation?

Reply:
ok for now. We may want to move the proto file into jaeger-idl in the future.